ynr/apps/elections/uk/templates/candidates/_candidates_for_post.html

```diff
@@ -1,59 +1,6 @@
 {% load standing %}
 
```

Comment:
Is the `_candidates_for_post.html` template actually used anywhere now? There's still a bunch of stuff in it, but I can't find anywhere its included. If not, can we delete it.